README.rst

```diff
@@ -28,7 +28,7 @@ processing and installation script knows how to produce the HTML
 for either PEP format.
 
 For processing reStructuredText format PEPs, you need the docutils
-package, which is available from `PyPI <http://pypi.python.org>`_.
+package, which is available from `PyPI <https://pypi.python.org>`_.
```

Comment:
This one redirects to:
```suggestion
package, which is available from `PyPI <https://pypi.org/>`_.
```

Reply:
Yeah, I didn't dare changing that, because I didn't know if there are some kind of ownership considerations to be taken into account.

But will commit the suggestion now :)

Reply:
Thanks!